### ISSUE TITLE
KeyPath observers not removed

### ISSUE THREAD

**Issue by ebaker355** [HarshilShah/DeckTransition#27]
Using version 1.0.0 installed via CocoaPods, when I dismiss a view controller that I presented with DeskTransition, I receive the following crash:

```
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'An instance 0x7fbb03e54680 of class UIView was deallocated while key value observers were still registered with it. Current observation info: <NSKeyValueObservationInfo 0x6080002373e0> (
<NSKeyValueObservance 0x608000243e40: Observer: 0x7fbb03c56370, Key path: frame, Options: <New: NO, Old: NO, Prior: NO> Context: 0x0, Property: 0x608000240120>
<NSKeyValueObservance 0x608000240660: Observer: 0x7fbb03c56370, Key path: transform, Options: <New: NO, Old: NO, Prior: NO> Context: 0x0, Property: 0x608000243750>
)'
```

I believe that these key path observations in `DeckPresentationController.swift` are never removed via `removeObserver(_:forKeyPath:)`:

```
presentedViewController.view.addObserver(self, forKeyPath: "frame", options: [.initial], context: nil)
presentedViewController.view.addObserver(self, forKeyPath: "transform", options: [.initial], context: nil)
```

Thanks for all of your hard work on this!

**Comment by HarshilShah** [comment]
Thank you for this find! For some reason I hadn't run into this in the example app or another app I'm working on. Fixed in 1.0.1, which should be out soon (already out on GitHub/Carthage, `pod trunk push` is processing as I type this).

Glad you like it! Feel free to post any further issues or requests on here!

**Comment by ebaker355** [comment]
👍
